app/views/me/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if current_user %>
+	<% current_user.attributes.each do |k, v| %>
```

Comment:
List the attributes from the oauth hash as key value pairs for the current user. This can be narrowed down to show only the name of the current user and then the option to 'shake it up', or any other design decisions we agree on.

Reply:
I am not fully sure why we would show all attributes including provider information in the view as well as created at and updated at information

Reply:
It was just to confirm that oauth was working. We will adjust the view to be the dropdown with distance and price latere

Reply:
Yeah we had discussed this yesterday, Those will not be there and we will adjust what we finally want on the view. This was just to make sure we had access to those attributes.